package.json

```diff
@@ -26,7 +26,7 @@
     "broccoli-merge-trees": "^3.0.2",
     "ember-auto-import": "^1.3.0",
     "ember-cli-babel": "^7.4.0",
-    "pretender": "~3.0.1",
+    "pretender": "^3.4.3",
```

Comment:
Why not `~3.4.3` ?

Reply:
Because using `^` says we should be able to use all Pretender versions up until 4.0.0 rather than `~` which will only allow up until `3.5.0`. As Pretender uses semver there shouldn't be any breaking changes until 4.0. Also all the other dependencies use `^`.

Reply:
I know but remember this is how we solved https://github.com/adopted-ember-addons/ember-data-factory-guy/issues/387. What I suggested would help keep it "under control". We can intervene and bump the version again if needed anyway. WDYT?

Reply:
Also same fix was made here previously https://github.com/adopted-ember-addons/ember-data-factory-guy/pull/354

Reply:
@herzzanu I'd prefer to leave it with `^` as I think it's better to have looser dependencies for host apps. Mirage.js also uses pretender with `^`. I know there's been issues in the past with stability but I do seem to remember there was a large chunk of refactoring happening at that time (they were introducing fetch into Pretender) and it appears things are a lot more stable now.

Reply:
Sounds good to me 👍